help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+---
+keywords: crm;CRM;crm destinations;Microsoft Dynamics 365;Microsoft Dynamics 365 crm destination
+title: Microsoft Dynamics 365 connection
+description: The Microsoft Dynamics 365 destination allows you to export your account data and activate it within Microsoft Dynamics 365 for your business needs.
+---
+
+# [!DNL Microsoft Dynamics 365] connection
+
+## Overview {#overview}
+
+[!DNL Microsoft Dynamics 365](https://dynamics.microsoft.com/en-us/) is a cloud-based business application platform that combines Enterprise Resource Planning (ERP) and Customer Relationship Management (CRM) along with productivity applications and AI tools, to bring end-to-end smoother and more controlled operations, better growth potential and reduced costs.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [Contact Entity Reference API](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1), which allows you to update identities within a segment into [!DNL Dynamics 365].
+
+[!DNL Dynamics 365] uses OAuth 2 with Authorization Grant as the authentication mechanism to communicate with the Contact Entity Reference API. Instructions to authenticate to your [!DNL Dynamics 365] instance are further below, in the [Authenticate to destination](#authenticate) section.
```

Comment:
Is Dynamics 365 an approved branding for it? If it's not used in the Microsoft docs, let's use [!DNL Microsoft Dynamics 365] instead.

Reply:
Yes it is even their [homepage](https://dynamics.microsoft.com/en-us/?cdn=disable) refers to it.